archey/__main__.py

```diff
@@ -63,8 +63,8 @@ class Entries(Enum):
     GPU = e_GPU
     RAM = e_RAM
     Disk = e_Disk
-    LAN_IP = e_LanIp
-    WAN_IP = e_WanIp
+    LAN_IP = e_LanIP
```

Comment:
Okay for class renaming, but I'd rather keep these entries names as before, at least for BC purposes.